sc/managers/configman/store/file/files.go

```
@@ -58,12 +58,12 @@ func (f *File) GetResource(ctx context.Context, meta *model.ResourceMeta) (*mode
 
 	module, ok := scConfig.Config[meta.Module]
 	if !ok {
-		return nil, fmt.Errorf("no resource found for %s - %s - %s", meta.Module, meta.Type, meta.Name)
+		return &model.ResourceObject{}, nil
 	}
 
 	moduleType, ok := module[meta.Type]
 	if !ok {
-		return nil, fmt.Errorf("no resource found for %s - %s - %s", meta.Module, meta.Type, meta.Name)
+		return &model.ResourceObject{}, nil
```

This should return an error. Error needs to be ignored only for lists